tests/Unit/Index/Service/ResultIndexerTest.php

```diff
@@ -41,18 +42,22 @@ class ResultIndexerTest extends TestCase
 
     /** @var QueueDispatcherInterface|MockObject */
     private $queueDispatcher;
+    /** @var AdvancedSearchChecker|MockObject  */
```

Comment:
PSR + our code style agreement

```suggestion

    /** @var AdvancedSearchChecker|MockObject  */
```